web/packages/demo/www/index.js

```diff
@@ -119,13 +119,13 @@ window.addEventListener("DOMContentLoaded", async () => {
     }
     sampleFileInputContainer.style.display = "inline-block";
 
-    const initialFile = new URLSearchParams(window.location.search).get("file");
+    const initialFile = new URL(window.location).searchParams.get("file");
     if (initialFile) {
```

Comment:
What is the difference here?

Reply:
Not much, it's shorter and more readable in my opinion.